rudder-webapp/SOURCES/rudder-upgrade

```diff
@@ -379,6 +380,21 @@ upgrade_masterfiles() {
 # Check and update config
 ################################################################################
 
+upgrade_conf_file() {
+
+  STEP="Upgrade /opt/rudder/etc/rudder-web.properties"
+  CONF_FILE="/opt/rudder/etc/rudder-web.properties"
+
+  # - 7.0.0~alpha1 : Rename property rudder.community.port into rudder.policy.distribution.port.cfengine
+  OLD_PROPERTY_NAME="rudder.community.port"
+  NEW_PROPERTY_NAME="rudder.policy.distribution.port.cfengine"
+  ATTRIBUTESET=$(grep -c "^${NEW_PROPERTY_NAME}[ \t]*=" ${CONF_FILE} || true)
+  if [ ${ATTRIBUTESET} -eq 0 ]; then
+    sed -i -e "s/${OLD_PROPERTY_NAME}/${NEW_PROPERTY_NAME}/" ${CONF_FILE}
```

Comment:
this will still leave 2 configuration line if there is one of each at the beginning

Reply:
yes, but that's not a problem since we take the new one first

Reply:
how do you know it's the new one ? they both have the same name after this

Reply:
I don't understand ?
If the user added `rudder.policy.distribution.port.cfengine`, then we have both `rudder.policy.distribution.port.cfengine` and `rudder.community.port` but the `if` prevent the sed; rudder takes `rudder.policy.distribution.port.cfengine` as expected. 
If the user didn't do anything (nominal case), then there is only `rudder.community.port`, the sed is done and rudder use the only value. 
If somehow the upgrade script doesn't run, then doesn't find `rudder.policy.distribution.port.cfengine`, so it fall back to `rudder.community.port`.

Reply:
aah i understood the if the other way around